ceph_deploy/mon.py

```diff
@@ -535,6 +528,26 @@ def make(parser):
 # Helpers
 #
 
+def get_mon_member(args, init_tags=False):
```

Comment:
I'd prefer this to be named get_mon_members(), since there can me many monitors.